modelling/src/main/java/org/axonframework/modelling/entity/annotation/AnnotatedEntityMessagingMetamodel.java

```diff
@@ -174,45 +177,42 @@ private AnnotatedEntityModel(
         this.messageTypeResolver = requireNonNull(messageTypeResolver, "The messageTypeResolver may not be null.");
         requireNonNull(concreteTypes, "The concreteTypes may not be null.");
         if (!concreteTypes.isEmpty()) {
-            this.entityModel = initializePolymorphicModel(entityType, concreteTypes);
+            this.delegateMetamodel = initializePolymorphicModel(entityType, concreteTypes);
```

Comment:
Nit: You could rename the factory methods to `initializePolymorphicMetamodel` and `initializeConcreteMetamodel`.

Reply:
You are correct. Done!